test/OpConvertPtrToU_narrowing.spvasm

```diff
@@ -0,0 +1,26 @@
+; RUN: spirv-as --target-env spv1.0 -o %t.spv %s
```

Comment:
Don't you need `REQUIRES: spirv-as` for the `.spvasm` files in this PR?

Reply:
Good point. Thanks!

Reply:
Okay, have pushed a new version.